src/main/java/com/risevision/bigquery/BigqueryCommonApi.java

```diff
@@ -0,0 +1,36 @@
+package com.risevision.bigquery;
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.bigquery.Bigquery;
+import com.google.api.services.bigquery.BigqueryScopes;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.logging.Logger;
+
+public class BigqueryCommonApi {
```

Comment:
@ahmedalsudani This common functionality is nicely broken out here.   Did you consider alternatives to extracting a parent class and using inheritance?